crm_lead_code/views/crm_lead_view.xml

```diff
@@ -48,7 +48,7 @@
         <field name="inherit_id" ref="crm.crm_case_kanban_view_leads" />
         <field name="arch" type="xml">
             <field name="name" position="before">
-                <field name="code" />&amp;nbsp;
+                <field name="code" /><span> </span>
```

Comment:
Hi, thanks for this PR.
IMHO  `<span/>` should be better

Reply:
`<span/>` does not add the extra SPACE.
Or is there a css class which we could apply?

Reply:
Ok found a solution: `<span class="p-1" />` 
thank you for your feedback.